api/graphql/schema.graphql

```diff
@@ -306,6 +306,22 @@ enum SalaryRange {
     _100K_PLUS
 }
 
+enum ClassificationRole{
+    TECHNICIAN
+    ANALYST
+    TEAM_LEADER
+    TECHNICAL_ADVISOR
+    SENIOR_ADVISOR
+    MANAGER
+}
+
+type ClassificationRoles{
+    id: ID!
+    key: KeyString!
+    name: LocalizedString
+    level: Int
+}
```

Comment:
So, its not good to have two entities in the schema with the same name. And if we follow the same patterns we have so far, we don't need an enum at all as part of this PR.

However... using an enum instead of a KeyString might actually make sense! Unlike DB-only key strings, it would make the key available to codegen for typescript checking and auto-completion... In this case, we wanted to use a database model (instead of a simple enum) in order to link it to other models, but we don't expect admins to be adding ClassificationRoles to the database on their own.

Reply:
Changed it. Thank you!